ihatemoney/web.py

```diff
@@ -699,7 +699,6 @@ def delete_bill(bill_id):
 
 @main.route("/<project_id>/edit/<int:bill_id>", methods=["GET", "POST"])
 def edit_bill(bill_id):
-    # FIXME: Test this bill belongs to this project !
```

Comment:
How did you test this?
Looking at the code, it seems correct, but I don't see a test for this either :)